### DISCUSSION TITLE
Re-Naming in the Subjective Logic : Idea and Truth.

### DISCUSSION THREAD

**Discussion by ghost** [systemphil/sphil#103] [Hegel]
My question/remark may sound a bit odd.

Let's have in mind the triadic organization of the Subjective Logic : 
1) Concept (moments of the concept, judgment, syllogism)
2) Object (mechanical, chemical, teleology)
3) Idea (life, cognition, absolute idea)

_For some reason_, I find interesting to shift a little bit the second part of this organization : 
1) Concept (moments of the concept, judgment, syllogism)
2) Idea (object, life, cognition)
3) Absolute Idea (a, b, c)

What I have done is simply that I have put the development of the Object in the section Idea, and I have given a separate section to the Absolute Idea.

— One first reaction may be to argue that the Object does not correspond to the Idea, inasmuch as the Idea is an **active** correspondance of the concept with its "exposition" (Darstellung) — an exposition that constitutes the Object — then, the Object cannot be part of the Idea. The Object is truth in itself ; the Idea is, at the minimum, truth **for itself**.

In that case, I may simply rename my sections : 
1) Concept
2) Truth (Object = truth in itself, Life = truth for itself, Cognition = truth in-and-for itself)
3) Absolute Idea

— A second reaction may be to argue that the reorganization disturbs the logical progression of the dialectics, which we will simply call here universal-particular-singular.

So this question becomes : does the series Concept-Truth-Absolute Idea constitute a legitimate "imprint" of the UPS dialectics ?

Well, the discussion on this topic could probably last very long. But, at first sight, I think that it works :
- The first section, Concept, still constitutes the Universal moment of the subjective Logic.
- Truth can be seen as the Particular moment of the subjective Logic, insofar as it always deals with the ob-ject.
- The Absolute Idea represents the Singular moment, where the object is now totally "sublated" in the universal activity.

***

If we accept these arguments as valid, we can say : assuming a neat triadic organization of the Logic, all I have done is a modification of the "zoom" (triadic level) on certain sections, without ever modifying the content.

Obviously, what I have made here could be made elsewhere, at least theoretically. 

But I don't think it would be a good idea , unless one has a good reasons for doing it, and I think that, in this case, the weighty notion of "Truth" offers such a reason to consider the new perspective as an interesting one.

**Comment by ridetoruin** [comment]
Interesting suggestion. 

I want to start off by just stating that I think the general topic of what names we assign to sections, chapters, and even determinations, is a topic of great interest to me and, I think, a topic that deserves greater attention because of the ramifications of the names that are assigned. 

My main concern with what you have suggested above is the idea of naming the second section "Truth". I understand why you're saying this, Hegel makes a lot of points about this in the introduction to the section on the Idea, but I think that this reveals a reading of the Logic, which I would find problematic. Setting aside what Hegel writes, why should we think that it is only in the second section of the Subjective Logic that we are talking about the Truth? What does that mean for previous sections of the Logic? Are they not true? If they aren't true, what does that tell us about the Idea, that it is based on an untrue things? 

I have more to say. But I would like to get your thoughts on the above for now.

I have also attached an excerpt from my chapter to the Idea where I write about this point. You'll see a more fleshed out position on how I accommodate Hegel's remarks about truth with the concerns I raised above about your reading

[Truth and The Idea.docx](https://github.com/user-attachments/files/17679478/Truth.and.The.Idea.docx)

**Comment by ghost** [comment]
Hello Ridetoruin

Thank you for your input. 

First it seems to me that your questioning may be applied to other categories as well : are not being, nothing, dasein, etc... all Concepts ? But wait, Hegel only deals with Concept very late ! Yet it is easier to deal with this paradox with Concept in mind, than it is with Truth, because there is no disagreement that all categories are concepts, whereas there is some "confusion" whether all categories are true (as I will try to develop below). 

So obviously, if Hegel unfolds the Concept after Dasein, it does not mean that Dasein is not a concept, but rather that Dasein is not yet Concept "for itself" but only "in itself". The same may be said about Truth : every category is true in itself, but not for itself. This is quite formal an answer, even though it does address the problem, and it catches up with your distinction between a general and a specific sense :  

> Hegel employs the word “true” in two different senses - there is the general sense that all necessary development is “true” because it is the development of the Concept in-itself, and the more specific sense of the unity of the Concept and Objectivity when the unity is adequate. 

Now it may be good to dwell a bit on this "confusion" about the truth of categories in general (whereas their being conceptual is not problematic).

The problem arises because every determination of thought always contains some kind of tension (or contradiction) and ends up being sublated. And this lurking argument : a contradictory category is not really "ad-equate", and thus not really true.

It seems to me that all the determinations are mainly thought by Hegel in the "becoming" mode, which indeed makes every category a potential "dead man", and builds the Absolute Truth as, so to speak, the System of Errors... 

However, the "conservation" aspect of the Aufhebung plays only a minor role in this unilaterally "temporal" understanding of sublation. Yet when Hegel compares poetically the determinations of thought with a diamond net, he is hinting at another, more spatial understanding. First, this comparison makes it clear that we use this net to think in general ; the presupposition-less exercise is but a conscious explicitation of this net. How odd it is to imagine that we would use falsities to think ! 

Second, the diamond net metaphor, in my opinion, allows to understand "conservation" in major mode. We can imagine that each category leaves a trace, that indeed "resists" becoming another category, so that each category does possess a unique "unbreakable kernel of truth" (please note that the meaning of diamond is precisely "unbreakable"). It is true (in it self), inasmuch as this "ash" is an objectivation of the thought process.

I like to compare each category with a musical note, and imagine that the Absolute Idea (to keep matters simple) is like a celtic harp (organized totality of strings). This is a spatialization of the temporally-generated categories. According to the diamond net comparison, to think is like to play music, using the right strings in the right order and so on. 

(By the way, this trace-leaving dialectics may be one way to dialogue with Derrida) 

In conclusion, I think that the confusion about the truth of categories in general, arises from a too unilateral understanding of Aufhebung, putting weight on the "temporal" suppression but forgetting the "unbreakable remains" of each sublation that make for its truth.
